common/utils/auth-utils.js

```diff
@@ -4,7 +4,7 @@ import { setAuthCookies, removeAuthCookies, hasValidAuthToken } from './cookie-u
 
 export const login = ({ token, user }, routeTo = '/profile') => {
   setAuthCookies({ token, user });
-  Router.push(routeTo);
+  return Router.push(routeTo);
```

Comment:
This yields a `Promise<boolean>` to let you know when the route change has successfully gone through.

Reply:
Ah. So then instead of return, when if we make this method async and then make this line the `await`?

(Same with join and login pages)

Reply:
Yes, that would produce the same result.

Reply:
Would you make that change? I like the explicitness of it.

Reply:
Sure, I can do that.